.github/workflows/deploy-support-func.yaml

```
@@ -4,10 +4,14 @@ on:
   workflow_dispatch: {}
 
 jobs:
-  deploy_workspace_to_azure:
+  deploy:
     name: Deploy
-    uses: pagopa/io-std/.github/workflows/deploy-workspace.yaml@main
-    with:
-      environment: io-p-sign-support-func
-      workspace-name: io-func-sign-support
+    uses: pagopa/dx/.github/workflows/function_app_deploy.yaml@main
     secrets: inherit
+    with:
+      workspace_name: io-func-sign-support
+      environment: webapp-prod-cd
```

```suggestion
      environment: webapp-prod
```

`cd` is appended by the template